lib/date_time/date_time_core.c

```diff
@@ -110,10 +148,12 @@ static void date_time_update_work_fn(struct k_work *work)
 		date_time_core_store(date_time_ms_ntp, DATE_TIME_OBTAINED_NTP);
 		return;
 	}
+
```

Comment:
Nit: Is this intentional new line. I'd remove it.

Reply:
I have a bad habit, sorry. Will remove.